wgengine/router/router_linux.go

```diff
@@ -163,11 +163,15 @@ func (r *linuxRouter) Close() error {
 }
 
 // Set implements the Router interface.
-func (r *linuxRouter) Set(cfg *Config) error {
```

Comment:
See https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

Let's not name it here, as it doesn't add anything but stutter.

Reply:
Oops, I forgot to removed that. Fixed!